include/Config.h

```diff
@@ -29,12 +29,12 @@ namespace mull {
 
 class Config {
   std::string bitcodeFileList;
+  std::string dynamicLibraryFileList;
```

Comment:
I don't like the fact that you moved `dynamicLibraryFileList` to be the second in order. `bitcodeFileList` goes first because it is the only required parameter in a config.yml. Then, as I see it, parameters more important than dynamicLibraryFileList should go. I would suggest you to move it where it was. But let's also listen to @AlexDenisov .